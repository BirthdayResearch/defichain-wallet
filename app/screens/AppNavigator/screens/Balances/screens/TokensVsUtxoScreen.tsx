import { MaterialIcons } from '@expo/vector-icons'
import React from 'react'
import { View } from 'react-native'
import { Text } from '../../../../../components'
import { translate } from '../../../../../translations'
import { tailwind } from '../../../../../tailwind'
import { getTokenIcon } from '../../../../../components/icons/tokens/_index'

export function TokensVsUtxoScreen (): JSX.Element {
  return (
    <View testID='token_vs_utxo_screen'>
      <View style={tailwind('pt-6 px-4 border-b-2 border-gray-200')}>
        <Text style={tailwind('text-black pb-6')}>
          {translate('screens/ConvertScreen', 'DFI exists in two forms â€“ UTXO and token, which are interchangeable depending on the usage. You can easily convert between the two forms with the app.')}
        </Text>
        <Text style={tailwind('text-black pb-6')}>
          {translate('screens/ConvertScreen', 'UTXO DFI is the main form DFI and is used for core cryptocurrency purposes like send, receive and fees. DFI tokens are used for DeFi functions such as swaps and liquidity mining.')}
        </Text>
        <Text style={tailwind('text-black pb-4')}>
          {translate('screens/ConvertScreen', 'Your DFI balance includes both UTXO and tokens.')}
        </Text>
      </View>
      <View style={tailwind('flex flex-row h-full')}>
        <View style={tailwind('w-6/12 border-r-2 border-gray-200 px-4 py-5')}>
          <ComparisonTitle tokenUnit='_UTXO' />
          <ComparisonRow label='Core crypto services' />
          <ComparisonRow label='Send to other wallets' />
          <ComparisonRow label='Receive tokens' />
          <ComparisonRow label='Service fees' />
        </View>
        <View style={tailwind('w-6/12 px-4 py-5')}>
          <ComparisonTitle tokenUnit='DFI' />
          <ComparisonRow label='DeFi services' />
          <ComparisonRow label='Liquidity mining' />
          <ComparisonRow label='Atomic Swaps' />
        </View>
      </View>
    </View>
  )
}

function ComparisonTitle (props: {tokenUnit: '_UTXO' | 'DFI'}): JSX.Element {
  const TokenIcon = getTokenIcon(props.tokenUnit)
  const label = props.tokenUnit === '_UTXO' ? 'DFI (UTXO)' : 'DFI (Token)'

  return (
    <View style={tailwind('flex flex-row items-center pb-5')}>
      <TokenIcon width={24} height={24} style={tailwind('mr-2')} />
      <Text style={tailwind('text-lg')}>{translate('screens/ConvertScreen', label)}</Text>
    </View>
  )
}

function ComparisonRow (props: {label: string}): JSX.Element {
  return (
    <View style={tailwind('flex flex-row pb-2')}>
      <MaterialIcons name='check' size={24} style={tailwind('text-green-500 mr-2')} />
      <Text style={tailwind('text-sm font-medium')}>{translate('screens/ConvertScreen', props.label)}</Text>
    </View>
  )
}
