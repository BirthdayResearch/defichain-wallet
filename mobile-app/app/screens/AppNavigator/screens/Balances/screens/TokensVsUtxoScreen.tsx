import React from 'react'
import { View } from 'react-native'
import { getNativeIcon } from '@components/icons/assets'
import { ThemedIcon, ThemedScrollView, ThemedText } from '@components/themed'
import { tailwind } from '@tailwind'
import { translate } from '@translations'

export function TokensVsUtxoScreen (): JSX.Element {
  return (
    <ThemedScrollView
      contentContainerStyle={{
        flexGrow: 1,
        justifyContent: 'space-between'
      }}
      style={tailwind('h-full')}
      testID='token_vs_utxo_screen'
    >
      <View style={tailwind('pt-6 px-4 border-b-2 border-gray-200')}>
        <ThemedText style={tailwind('pb-6')}>
          {translate('screens/TokensVsUtxoScreen', 'DFI exists in two forms â€“ UTXO and token, which are interchangeable depending on the usage. You can easily convert between the two forms with the app.')}
        </ThemedText>

        <ThemedText style={tailwind('pb-6')}>
          {translate('screens/TokensVsUtxoScreen', 'UTXO DFI is the main form DFI and is used for core cryptocurrency purposes like send, receive and fees. DFI tokens are used for DeFi functions such as swaps and liquidity mining.')}
        </ThemedText>

        <ThemedText style={tailwind('pb-4')}>
          {translate('screens/TokensVsUtxoScreen', 'Your DFI balance includes both UTXO and tokens.')}
        </ThemedText>
      </View>

      <View style={tailwind('flex flex-row flex-grow self-stretch items-stretch')}>
        <View style={tailwind('w-6/12 border-r-2 border-gray-200 px-4 py-5')}>
          <ComparisonTitle tokenUnit='_UTXO' />

          <ComparisonRow label='Core crypto services' />

          <ComparisonRow label='Send to other wallets' />

          <ComparisonRow label='Receive tokens' />

          <ComparisonRow label='Service fees' />
        </View>

        <View style={tailwind('w-6/12 px-4 py-5')}>
          <ComparisonTitle tokenUnit='DFI' />

          <ComparisonRow label='DeFi services' />

          <ComparisonRow label='Liquidity mining' />

          <ComparisonRow label='Atomic Swaps' />
        </View>
      </View>
    </ThemedScrollView>
  )
}

function ComparisonTitle (props: {tokenUnit: '_UTXO' | 'DFI'}): JSX.Element {
  const TokenIcon = getNativeIcon(props.tokenUnit)
  const label = props.tokenUnit === '_UTXO' ? 'DFI (UTXO)' : 'DFI (Token)'

  return (
    <View style={tailwind('flex flex-row items-center pb-5')}>
      <TokenIcon style={tailwind('mr-2')} />

      <ThemedText style={tailwind('text-lg')}>
        {translate('screens/TokensVsUtxoScreen', label)}
      </ThemedText>
    </View>
  )
}

function ComparisonRow (props: {label: string}): JSX.Element {
  return (
    <View style={tailwind('flex flex-row pb-2')}>
      <ThemedIcon
        dark={tailwind('text-darksuccess-500')}
        iconType='MaterialIcons'
        light={tailwind('text-success-500')}
        name='check'
        size={24}
        style={tailwind('mr-2')}
      />

      <ThemedText
        style={tailwind('text-sm font-medium flex-1')}
      >
        {translate('screens/TokensVsUtxoScreen', props.label)}
      </ThemedText>
    </View>
  )
}
