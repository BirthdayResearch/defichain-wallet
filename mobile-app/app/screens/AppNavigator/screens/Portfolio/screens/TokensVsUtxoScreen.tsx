
import { View } from 'react-native'
import { getNativeIcon } from '@components/icons/assets'
import { ThemedIcon, ThemedScrollView, ThemedText } from '@components/themed'
import { tailwind } from '@tailwind'
import { translate } from '@translations'

export function TokensVsUtxoScreen (): JSX.Element {
  return (
    <ThemedScrollView
      contentContainerStyle={tailwind('p-6 pb-8')}
      testID='token_vs_utxo_screen'
    >
      <View style={tailwind('pb-6')}>
        <ThemedText style={tailwind('pb-2 text-lg font-semibold')}>
          {translate('screens/TokensVsUtxoScreen', 'Learn about UTXOs and Tokens')}
        </ThemedText>

        <ThemedText
          light={tailwind('text-gray-700')}
          dark={tailwind('text-gray-200')}
        >
          {translate('screens/TokensVsUtxoScreen', 'DFI exists in two forms - UTXO and Token. They can be converted with one another to serve different use cases within your light wallet.')}
        </ThemedText>
      </View>
      <FaqInfoSection
        title='DFI in UTXO form'
        description='DFI in UTXO form is the primary form of DFI. It is used for core cryptocurrency purposes such as send, receive and fees.'
        tokenUnit='_UTXO'
        useCases={[
          'Sending DFI to other wallets',
          'Receiving DFI from other wallets',
          'Transaction fees'
        ]}
      />
      <FaqInfoSection
        title='DFI in token form'
        description='DFIs in token form are used for various DeFiChain functions such as Liquidity Pools, Loans, Auctions and DEX swaps.'
        tokenUnit='DFI'
        useCases={[
          'Swapping DFI with other tokens (e.g. dBTC, dETH, dUSDT)',
          'Liquidity pools',
          'Auctions and Loans'
        ]}
      />
    </ThemedScrollView>
  )
}

interface FaqInfoSectionProps {
  title: string
  description: string
  tokenUnit: '_UTXO' | 'DFI'
  useCases: string[]
}

function FaqInfoSection (props: FaqInfoSectionProps): JSX.Element {
  const TokenIcon = getNativeIcon(props.tokenUnit)
  return (
    <View style={tailwind('mb-6')}>
      <View style={tailwind('flex-row items-center flex-grow mb-2')}>
        <TokenIcon width={24} height={24} />
        <View style={tailwind('ml-2 flex-auto')}>
          <ThemedText
            dark={tailwind('text-gray-200')}
            light={tailwind('text-black')}
            style={tailwind('text-lg font-semibold')}
          >
            {translate('screens/TokensVsUtxoScreen', props.title)}
          </ThemedText>
        </View>
      </View>
      <View style={tailwind('mb-4')}>
        <ThemedText
          light={tailwind('text-gray-700')}
          dark={tailwind('text-gray-200')}
        >
          {translate('screens/TokensVsUtxoScreen', props.description)}
        </ThemedText>
      </View>
      <View style={tailwind('mb-2')}>
        <ThemedText
          light={tailwind('text-gray-700')}
          dark={tailwind('text-gray-200')}
          style={tailwind('text-sm')}
        >
          {translate('screens/TokensVsUtxoScreen', 'Use cases include:')}
        </ThemedText>
      </View>
      {props.useCases.map((label) => <UseCaseRow key={label} label={label} />)}
    </View>
  )
}

function UseCaseRow (props: {label: string}): JSX.Element {
  return (
    <View style={tailwind('flex-row items-start flex-grow mb-2')}>
      <ThemedIcon
        size={16}
        name='check-circle'
        iconType='MaterialIcons'
        light={tailwind('text-success-500')}
        dark={tailwind('text-darksuccess-500')}
        style={tailwind('mt-0.5')}
      />
      <View style={tailwind('ml-2 flex-auto')}>
        <ThemedText
          light={tailwind('text-gray-700')}
          dark={tailwind('text-gray-200')}
          style={tailwind('text-sm')}
        >
          {translate('screens/TokensVsUtxoScreen', props.label)}
        </ThemedText>
      </View>
    </View>
  )
}
