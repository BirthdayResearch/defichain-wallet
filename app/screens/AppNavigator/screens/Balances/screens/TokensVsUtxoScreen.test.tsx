import { render } from '@testing-library/react-native'
import * as React from 'react'
import { TokensVsUtxoScreen } from './TokensVsUtxoScreen'

jest.mock('../../../../../contexts/ThemeProvider')
describe('tokens vs utxo screen', () => {
  it('should render', async () => {
    const component = (
      <TokensVsUtxoScreen />
    )
    const rendered = render(component)
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
