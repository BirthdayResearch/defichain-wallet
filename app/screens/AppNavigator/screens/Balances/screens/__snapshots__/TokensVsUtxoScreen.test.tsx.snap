// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokens vs utxo screen should render 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "flexGrow": 1,
      "justifyContent": "space-between",
    }
  }
  style={
    Array [
      Object {
        "height": "100%",
      },
      Object {
        "backgroundColor": "rgba(245, 245, 245, 1)",
      },
    ]
  }
  testID="token_vs_utxo_screen"
>
  <View>
    <View
      style={
        Object {
          "borderBottomColor": "rgba(229, 229, 229, 1)",
          "borderBottomWidth": 2,
          "borderLeftColor": "rgba(229, 229, 229, 1)",
          "borderRightColor": "rgba(229, 229, 229, 1)",
          "borderTopColor": "rgba(229, 229, 229, 1)",
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 24,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "RegularFont",
              "fontSize": 16,
              "fontWeight": "400",
              "lineHeight": 24,
            },
            Array [
              Object {
                "paddingBottom": 24,
              },
              Object {
                "color": "rgba(0, 0, 0, 1)",
              },
            ],
          ]
        }
      >
        DFI exists in two forms â€“ UTXO and token, which are interchangeable depending on the usage. You can easily convert between the two forms with the app.
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "RegularFont",
              "fontSize": 16,
              "fontWeight": "400",
              "lineHeight": 24,
            },
            Array [
              Object {
                "paddingBottom": 24,
              },
              Object {
                "color": "rgba(0, 0, 0, 1)",
              },
            ],
          ]
        }
      >
        UTXO DFI is the main form DFI and is used for core cryptocurrency purposes like send, receive and fees. DFI tokens are used for DeFi functions such as swaps and liquidity mining.
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "RegularFont",
              "fontSize": 16,
              "fontWeight": "400",
              "lineHeight": 24,
            },
            Array [
              Object {
                "paddingBottom": 16,
              },
              Object {
                "color": "rgba(0, 0, 0, 1)",
              },
            ],
          ]
        }
      >
        Your DFI balance includes both UTXO and tokens.
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "stretch",
          "alignSelf": "stretch",
          "display": "flex",
          "flexDirection": "row",
          "flexGrow": 1,
        }
      }
    >
      <View
        style={
          Object {
            "borderBottomColor": "rgba(229, 229, 229, 1)",
            "borderLeftColor": "rgba(229, 229, 229, 1)",
            "borderRightColor": "rgba(229, 229, 229, 1)",
            "borderRightWidth": 2,
            "borderTopColor": "rgba(229, 229, 229, 1)",
            "paddingBottom": 20,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 20,
            "width": "50%",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 20,
            }
          }
        >
          <RNSVGSvgView
            bbHeight={32}
            bbWidth={32}
            focusable={false}
            height={32}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "marginRight": 8,
                },
                Object {
                  "flex": 0,
                  "height": 32,
                  "width": 32,
                },
              ]
            }
            width={32}
          >
            <RNSVGGroup>
              <RNSVGCircle
                cx={16}
                cy={16}
                fill={4294901935}
                propList={
                  Array [
                    "fill",
                  ]
                }
                r={16}
              />
              <RNSVGPath
                d="M18.217 23.454V8.546c3.21.958 5.555 3.937 5.555 7.454s-2.346 6.496-5.555 7.454M15.994 6v8.428l-1.269-1.27-.162-3.167 1.324-3.985a9.952 9.952 0 00-2.454.338l-.64 1.926-1.816-.91a10.03 10.03 0 00-1.974 1.497l3.363 1.685.098 1.928-1.927-.099L8.85 9.01a10.035 10.035 0 00-1.497 1.973l.911 1.816-1.927.64A9.978 9.978 0 006 15.892l3.986-1.324 3.168.162 1.27 1.27-1.27 1.27-3.168.162L6 16.108c.009.849.13 1.669.338 2.454l1.927.64-.911 1.816c.418.719.92 1.383 1.497 1.973l1.686-3.362 1.927-.099-.098 1.928-3.363 1.685a10.04 10.04 0 001.974 1.498l1.815-.911.64 1.926a9.94 9.94 0 002.455.338l-1.324-3.985.162-3.168 1.27-1.27V26c5.522 0 10-4.477 10-10s-4.478-10-10-10"
                fill={4294967295}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontSize": 18,
                    "lineHeight": 28,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            DFI (UTXO)
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Core crypto services
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Send to other wallets
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Receive tokens
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Service fees
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingBottom": 20,
            "paddingLeft": 16,
            "paddingRight": 16,
            "paddingTop": 20,
            "width": "50%",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 20,
            }
          }
        >
          <RNSVGSvgView
            bbHeight={32}
            bbWidth={32}
            focusable={false}
            height={32}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "marginRight": 8,
                },
                Object {
                  "flex": 0,
                  "height": 32,
                  "width": 32,
                },
              ]
            }
            width={32}
          >
            <RNSVGGroup>
              <RNSVGCircle
                cx={16}
                cy={16}
                fill={4294954223}
                propList={
                  Array [
                    "fill",
                  ]
                }
                r={16}
              />
              <RNSVGPath
                d="M18.217 23.454V8.546c3.21.958 5.555 3.937 5.555 7.454s-2.346 6.496-5.555 7.454M15.994 6v8.428l-1.269-1.27-.162-3.167 1.324-3.985a9.952 9.952 0 00-2.454.338l-.64 1.926-1.816-.91a10.03 10.03 0 00-1.974 1.497l3.363 1.685.098 1.928-1.927-.099L8.85 9.01a10.035 10.035 0 00-1.497 1.973l.911 1.816-1.927.64A9.978 9.978 0 006 15.892l3.986-1.324 3.168.162 1.27 1.27-1.27 1.27-3.168.162L6 16.108c.009.849.13 1.669.338 2.454l1.927.64-.911 1.816c.418.719.92 1.383 1.497 1.973l1.686-3.362 1.927-.099-.098 1.928-3.363 1.685a10.04 10.04 0 001.974 1.498l1.815-.911.64 1.926a9.94 9.94 0 002.455.338l-1.324-3.985.162-3.168 1.27-1.27V26c5.522 0 10-4.477 10-10s-4.478-10-10-10"
                fill={4294901935}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontSize": 18,
                    "lineHeight": 28,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            DFI (Token)
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            DeFi services
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Liquidity mining
          </Text>
        </View>
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "paddingBottom": 8,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "fontFamily": "RegularFont",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
                Array [
                  Object {
                    "fontFamily": "MediumFont",
                    "fontSize": 14,
                    "fontWeight": "500",
                    "lineHeight": 20,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 1)",
                  },
                ],
              ]
            }
          >
            Atomic Swaps
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
